### PR TITLE
Toggle active org 418

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Relies on https://github.com/postgres-plugin/people/pull/39 to be merged

Relates to #418

Does not cover disabling a linked user when they are linked to an org that is being disabled.

@mk4111 please review and assign to the boss when done.

### CODE REVIEW

**Comment by SimonLab** [comment]
@JMurphyWeb do you know why the tests are failing? Reassign when fixed, thanks

**Comment by JMurphyWeb** [comment]
@SimonLab it says in the description.

File: server/handlers/orgs/toggle-active.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var Hoek = require('hoek');
+var helpers = require('../helpers.js');
```
Comment (SimonLab) [inline server/handlers/orgs/toggle-active.js]: "helpers" not used I think

Comment (JMurphyWeb) [inline server/handlers/orgs/toggle-active.js]: sorry. updated.

**Pull request merged**